engine/physics/src/physics/box2d/box2d_physics.cpp

```diff
@@ -813,42 +813,39 @@ namespace dmPhysics
         return shape_data;
     }
 
-    static b2Vec2 ComputeCentroid(const b2Vec2* vs, int count)
+    // From Box2dv3 (geometry.c)
+    static b2Vec2 b2ComputePolygonCentroid( const b2Vec2* vertices, int count )
```

Comment:
This function was copied from Box2d v2, so I now updated with the one from Box2d v3.